### DISCUSSION TITLE
RunOnce in case of reboot

### DISCUSSION THREAD

**Discussion by stanojevicmarko** [Squirrel/Squirrel.Windows#1844] [Q&A]
Hello,

I know reboot is not what we want to happen, but our app needs some prerequisites which on some machines ask for reboot. If this happens, installer should be called again after reboot. So, we were thinking of RunOnce registry, but custom events happen to late, as earliest is when installation is finished. Is there event, or how to hook into installation start? Or is there another solution?

Thx

**Comment by anaisbetts** [comment]
If you need this feature, Squirrel is likely not a good solution for you. Any prerequisite that requires elevation or reboot will not work because you will not be able to clean them up on uninstall.
